crt_portal/cts_forms/locale/es/LC_MESSAGES/django.po

```diff
@@ -703,7 +702,7 @@ msgstr "Delaware"
 
 #: model_variables.py:251
 msgid "District of Columbia"
-msgstr "Distrito de Columbia"
+msgstr "District of Columbia"
```

Comment:
double checking that these reversions to English were wanted?

Reply:
From the updated translation spreadsheet I believe so. Can confirm at stand-up